server/libs/userlibs.js

```diff
@@ -42,15 +42,55 @@ const register = function(userData) {
     })
 };
 
-const logIn = function(body) {
+const createToken = function(token, it) {
```

Comment:
Create token should be in the Token libs not user libs

Reply:
Done.